critRouter.js

```diff
@@ -27,14 +41,58 @@ router.get('/:crit_id',(req, res) => {
 				username: '@' + results[0].username
 			},
 			crit: {
+                replyCrits: [],
 				id: results[0].id,
 				created_on: results[0].created_on,
-				likes: 0,
-				replies: 0,
+				likes: results[0].likes,
+				replies: results[0].replies,
 				message: results[0].message
 			}
 		};
-		res.render('crit', crit);
+        let replies_query=`SELECT 
+        crits.id, crits.user_id, users.display_name, 
+        users.username, crits.crit_reply_id,
+        crits.message, crits.created_on,
+        count(crit_replies.id) AS replies,
+        ifnull(
+            (
+                SELECT count(crit_likes.id) 
+                FROM crit_likes 
+                WHERE crit_likes.crit_id = crits.id 
+                GROUP BY crit_likes.crit_id
+            ), 0
+        ) AS likes
+    FROM crits 
+    LEFT JOIN crits AS crit_replies
+        ON crit_replies.crit_reply_id = crits.id 
+    LEFT JOIN crit_likes
+        ON crit_likes.crit_id = crits.id
+    LEFT JOIN users 
+    ON crits.user_id = users.id 
+    WHERE crits.crit_reply_id = ?
+    GROUP BY crits.id`;
+        connection.query(replies_query, req.params.crit_id, (err, result) =>{
+
+            for(let i=0; i<result.length; i++){
```

Comment:
Consider adding a little bit of spacing around the words in your for loop so that it looks cleaner:

```suggestion
            for ( let i = 0; i < result.length; i++ ) {
```